File: javascript/negativos.js
const n = parseInt(prompt("Quantos numeros voce vai digitar?"));

let vetor = [], qtdNegativos = 0;

for(i = 0; i < n; i++) {
    vetor[i] = parseInt(prompt("Digite um numero:"));
}

console.log("NUMEROS NEGATIVOS:\n");

for(i = 0; i < n; i++) {
    if (vetor[i] < 0) {
        console.log(vetor[i]);
    }
}

File: python/negativos.py
n = int(input("Quantos numeros voce vai digitar? "))

vetor = []

for i in range(0, n):
    vetor.append(int(input("Digite um numero: ")))

print("NUMEROS NEGATIVOS:")
for i in range(0, n):
    if vetor[i] < 0:
        print(vetor[i])
